api_feature.go

```diff
@@ -62,6 +62,18 @@ func (f *APIFeature) RegisterSteps(ctx *godog.ScenarioContext) {
 	ctx.Step(`^I should receive the following response:$`, f.IShouldReceiveTheFollowingResponse)
 	ctx.Step(`^I should receive the following JSON response:$`, f.IShouldReceiveTheFollowingJSONResponse)
 	ctx.Step(`^I should receive the following JSON response with status "([^"]*)":$`, f.IShouldReceiveTheFollowingJSONResponseWithStatus)
+	ctx.Step(`^I use a service auth token "([^"]*)"$`, f.IUseAServiceAuthToken)
+	ctx.Step(`^I use an X Florence user token "([^"]*)"$`, f.IUseAnXFlorenceUserToken)
+}
+
+func (f *APIFeature) IUseAServiceAuthToken(serviceAuthToken string) error {
+	f.ISetTheHeaderTo("Authorization", serviceAuthToken)
```

Comment:
The service auth token has a `"Bearer "` prefix.

Reply:
added now